resources/linux/rpm/code.spec.template

```diff
@@ -55,24 +58,21 @@ ln -sf /usr/share/@@NAME@@/bin/@@NAME@@ %{_bindir}/@@NAME@@
 #fi
 
 # Update mimetype database to pickup workspace mimetype
-update-mime-database /usr/share/mime &> /dev/null || :
+update-mime-database %{_datadir}/mime &> /dev/null || :
 
 %postun
-if [ $1 = 0 ]; then
-  rm -f /usr/bin/@@NAME@@
-fi
-
 # Update mimetype database for removed workspace mimetype
-update-mime-database /usr/share/mime &> /dev/null || :
+update-mime-database %{_datadir}/mime &> /dev/null || :
 
 %files
 %defattr(-,root,root)
-%attr(4755, root, root) /usr/share/@@NAME@@/chrome-sandbox
+%attr(4755, root, root) %{_datadir}/%{name}/chrome-sandbox
 
-/usr/share/@@NAME@@/
-/usr/share/applications/@@NAME@@.desktop
-/usr/share/applications/@@NAME@@-url-handler.desktop
-/usr/share/mime/packages/@@NAME@@-workspace.xml
-/usr/share/pixmaps/@@ICON@@.png
-/usr/share/bash-completion/completions/@@NAME@@
-/usr/share/zsh/site-functions/_@@NAME@@
+%{_bindir}/%{name}
+%{_datadir}%{name}/
+%{_datadir}applications/%{name}.desktop
+%{_datadir}applications/%{name}-url-handler.desktop
+%{_datadir}mime/packages/%{name}-workspace.xml
+%{_datadir}pixmaps/@@ICON@@.png
+%{_datadir}bash-completion/completions/%{name}
+%{_datadir}zsh/site-functions/_%{name}
```

Comment:
Is the path separator not required between the macro and constants ? Seems inconsistent with other usages in this file.
```suggestion
%{_datadir}/%{name}/
%{_datadir}/applications/%{name}.desktop
%{_datadir}/applications/%{name}-url-handler.desktop
%{_datadir}/mime/packages/%{name}-workspace.xml
%{_datadir}/pixmaps/@@ICON@@.png
%{_datadir}/bash-completion/completions/%{name}
%{_datadir}/zsh/site-functions/_%{name}
```

Reply:
Actually, that's a good point. You're right -- `%{buildroot}` is by convention used without a trailing slash (so that it doesn't become the root directory if it's unset). But `%{_datadir}` should be followed by a slash, since at worst it's redundant. Thanks!